controllers/container.go

```diff
@@ -14,13 +14,15 @@ type ContainerController struct {
 // @Title Get
 // @Description find object by objectid
 // @Param	containerId		path 	string	true		"the containerId you want to get"
+// @Param	tail		path 	string	false		"number of lines you want to get"
 // @Success 200 {container} models.Container
 // @Failure 403 :containerID is empty
-// @router /:containerId/logs [get]
+// @router /:containerId/logs/:tail [get]
```

Comment:
Tail should be a query param. Tail is not an `id` of the collection `logs`. It's a way to filter the collection.

Also this would match the docker api https://docs.docker.com/engine/api/v1.30/#operation/ContainerLogs